certified-connectors/OutlookTasks/apiDefinition.swagger.json

```diff
@@ -363,22 +678,920 @@
         "Name": "test group"
       }
     },
-    "@odata.nextLink": {
-      "type": "string",
-      "x-ms-visibility": "advanced",
-      "title": "OData nextLink"
-    }
-  },
-  "parameters": {
-    "OdataQueryOption-$top": {
-      "in": "query",
-      "name": "$top",
-      "type": "integer",
-      "required": false,
-      "minimum": 0,
-      "x-ms-visibility": "advanced",
-      "description": "The number of items in the queried collection to be included in the result.",
-      "x-ms-summary": "Top Items"
+    "TaskOS": {
+      "description": "An Outlook task object.",
+      "type": "object",
+      "required": [
+        "Subject"
+      ],
+      "properties": {
+        "@odata.id": {
+          "type": "string",
+          "readOnly": true,
+          "x-ms-visibility": "internal"
+        },
+        "@odata.etag": {
+          "type": "string",
+          "readOnly": true,
+          "x-ms-visibility": "internal"
+        },
+        "Id": {
+          "title": "Id",
+          "description": "Unique identifier of the task.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-visibility": "important"
+        },
+        "ChangeKey": {
+          "type": "string",
+          "readOnly": true,
+          "x-ms-visibility": "internal"
+        },
+        "ParentFolderId": {
+          "title": "Parent Folder Id",
+          "description": "The unique id of the folder in which this task is located.",
+          "type": "string",
+          "readOnly": true
+        },
+        "AssignedTo": {
+          "title": "Assigned To",
+          "description": "Name of the person who has been assigned the task.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-visibility": "advanced"
+        },
+        "Owner": {
+          "title": "Owner",
+          "description": "Name of the person who created the task.",
+          "type": "string",
+          "readOnly": true,
+          "x-ms-visibility": "advanced"
+        },
+        "CreatedDateTime": {
+          "title": "Created",
+          "description": "The date and time when the task was created.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "x-ms-visibility": "advanced"
+        },
+        "LastModifiedDateTime": {
+          "title": "Last Modified Date-Time",
+          "description": "The date and time when this task was last modified.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "x-ms-visibility": "advanced"
+        },
+        "HasAttachments": {
+          "title": "Has attachments",
+          "description": "True if this task has attachments.",
+          "type": "boolean",
+          "readOnly": true,
+          "x-ms-visibility": "advanced"
+        },
+        "Status": {
+          "$ref": "#/definitions/TaskOS-Status"
```

Comment:
Does TaskOS stand for TaskOpenSource? Why are we making this distinction?

Reply:
Yes, it means TaskOS. The reason for the distinction is that we have existing operations in our AAPT Repo which use the same name. We need to make sure that our open-source type definitions do not conflict with the non-open sourced ones. Esp since there are breaking changes between them.